### DISCUSSION TITLE
Is it possible to execute tr3nch from rigtools v127?

### DISCUSSION THREAD

**Discussion by Balistic123** [3kh0/ext-remover#1219] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
Is it possible to execute tr3nch from rigtools v127? im on asus chromebook flip c434ta version 127.0.6533.138 shyvana

**Comment by CGRmags** [comment]
shut up will ya

On Mon, Oct 28, 2024 at 6:06 AM Balistic123 ***@***.***>
wrote:

> Is it possible to execute tr3nch from rigtools v127? im on asus chromebook
> flip c434ta version 127.0.6533.138 shyvana
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/3kh0/ext-remover/discussions/1219>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BDXUJBD6MPMNS2ZCWNWV4O3Z5YZGPAVCNFSM6AAAAABQXMBSGCVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZXGM4DKMRTG4>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>

-- 
NOTICE: This email message and/or its attachments may contain information 
that is confidential or restricted. It is intended only for the individuals 
named as recipients in the message. If you are NOT an authorized recipient, 
you are prohibited from using, delivering, distributing, printing, copying, 
or disclosing the message or content to others and must delete the message 
from your computer. If you have received this message in error, please 
notify the sender by return email.

**Comment by HelloIloveGimkit2** [comment, marked as answer]
there is a method @CGRmags is retarded basically you just use the method on the readme here's an explanation

if the extensions manifest has these things:
- tabs 
- activeTab 
- browserAction 
- 'unsafe-eval' set in the CSP (content-security-policy)
all of these are in the manifest and run this code if it has these permissions
```js
chrome.browserAction.onClicked.addListener(() => {chrome.tabs.executeScript(null, {code: `location.href="javascript:replace this part with your own bookmarklet.";`});});
```
